src/main/java/com/spaceclub/form/service/FormService.java

```diff
@@ -75,8 +74,20 @@ public void createApplicationForm(Long userId, Long eventId, List<FormOptionUser
     }
 
 
-    public List<FormApplicationGetResponse> getAllForms() {
-        return List.of(FormApplicationGetResponse.builder().build());
+    public FormApplicationGetInfo getApplicationForms(Long userId, Long eventId) {
+        Event event = validateEventAndCheckFormValidation(eventId);
+        Form form = event.getForm();
+
+        List<FormOptionUser> formOptionUsers = formOptionUserRepository.findByFormOption_Form_Id(form.getId());
+
+        return new FormApplicationGetInfo(form, formOptionUsers, event.getEventUsers());
+    }
+
+    private Event validateEventAndCheckFormValidation(Long eventId) {
```

Comment:
```suggestion
    private Event validateEventAndForm(Long eventId) {
```
이정도로 메소드 이름을 줄여도 의미전달이 충분히 될 것 같은데 어떠신가요?